content/300-guides/300-upgrade-guides/03-upgrade-prisma-migrate.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Upgrade to Prisma Migrate from legacy Prisma Migrate'
+metaTitle: 'Upgrade from legacy Prisma Migrate to the most recent Prisma Migrate.'
+---
+
+<TopBlock>
+
+The following guide describes how to upgrade from legacy Prisma Migrate (an Experimental feature available until version 2.12.0) to Prisma Migrate (2.13.0 and later).
+
+</TopBlock>
+
+## Prerequisites and assumptions
+
+This guide assumes that you have already added Prisma to your project.
+
+## Upgrade development environments
+
+1. Make sure you Prisma schema is in sync with your database schema. To make sure, introspect your database:
+
+   ```terminal
+   npx prisma introspect
+   ```
+
+2. Initialize a new migration history:
+
+   ```terminal
+   npx prisma migrate dev
+   ```
+
+   Prisma will perform the following steps:
+
+   1. Create a new migration directory under `prisma/migrations` with SQL migration file in it - for example: `20201126125825_first_migration/migration.sql`
+   1. Detect 
+
+3. **Initialize a new migration history** by calling `prisma migrate dev`. Prisma Migrate will do the following
```

Comment:
```suggestion
3. **Initialize a new migration history** by calling `prisma migrate dev --preview-feature`. Prisma Migrate will do the following
```
The `--preview-feature` is required for all migrate commands until Migrate is out of Preview